src/components/FAQ/FAQSections.tsx

```diff
@@ -0,0 +1,7 @@
+import { Accordion, AccordionProps } from "@chakra-ui/react";
+import type { FunctionComponent } from "react";
+
+export const FAQSections: FunctionComponent<AccordionProps> = ({
+  children,
+  ...accordionProps
+}) => <Accordion {...accordionProps}>{children}</Accordion>;
```

Comment:
I wonder if it would make sense to make "allowMultiple" enabled by default in this component?